client/src/controllers/OrderableController.ts

```diff
@@ -10,30 +14,38 @@ enum Direction {
  * Enables the ability for drag & drop or manual re-ordering of elements
  * within a prescribed container or the controlled element.
  *
- * Once re-ordering is completed an async request will be made to the
- * provided URL to submit the update per item.
+ * If the url value is provided, the controller will submit the updated
+ * order to the server via an async POST request once re-ordering is
+ * completed (via drag & drop) or manually via calling the submit method.
+ * This allows for granular keyboard control without submitting to the server
+ * every change.
  *
  * @example
  * ```html
  * <fieldset data-controller="w-orderable" data-w-orderable-url-value="/path/to/orderable/">
- *   <input type="button" data-w-orderable-target="item" data-w-orderable-item-id="1" value="Item 1"/>
- *   <input type="button" data-w-orderable-target="item" data-w-orderable-item-id="2" value="Item 2"/>
- *   <input type="button" data-w-orderable-target="item" data-w-orderable-item-id="3" value="Item 3"/>
+ *  <input type="button" data-w-orderable-target="item" data-w-orderable-item-id="1" value="Item 1"/>
```

Comment:
I think this nesting is a typo, probably from a previous merge issue sory..
It should be nested as it was.